### PR TITLE
Add support for RP432MP panel

### PR DESCRIPTION
**Pull request by maartenweyns**
As discussed in #9, this PR adds support for the new Risco RW432MP panel (e.g. LightSys+).

### CODE REVIEW

File: pyrisco/local/panels.py [hunk 1]
```diff
@@ -47,6 +47,18 @@ def _rw432_capabilities(firmware):
...
 
+def _rw432mp_capabilities(firmware):
+  max_zones = 512
+  max_outputs = 32
+
+  return {
+    PANEL_MODEL: 'LightSys+',
+    MAX_ZONES: max_zones,
+    MAX_PARTS: 32,
+    MAX_OUTPUTS: max_outputs,
```
Comment (OnFreund) [inline pyrisco/local/panels.py]:
```suggestion
    MAX_ZONES: 512,
    MAX_PARTS: 32,
    MAX_OUTPUTS: 32,
```

File: pyrisco/local/panels.py [hunk 2]
```diff
@@ -47,6 +47,18 @@ def _rw432_capabilities(firmware):
     MAX_OUTPUTS: max_outputs,
   }
 
+def _rw432mp_capabilities(firmware):
+  max_zones = 512
+  max_outputs = 32
```
Comment (OnFreund) [inline pyrisco/local/panels.py]:
```suggestion
```

File: pyrisco/local/panels.py (hunk 1 above)
Comment (OnFreund) [reply pyrisco/local/panels.py]: Also, shouldn't max outputs be 196?

File: pyrisco/local/panels.py [hunk 3]
```diff
@@ -69,6 +81,7 @@ def _rw512_capabilities(firmware):
   'RW232': _rw232_capabilities,
   'RW332': _rw332_capabilities,
   'RP432': _rw432_capabilities,
+  'RP432MP': _rw432mp_capabilities,
```
Comment (OnFreund) [inline pyrisco/local/panels.py]: shouldn't it be `_rp432mp_capabilities`?

File: pyrisco/local/risco_socket.py [hunk 4]
```diff
@@ -33,7 +33,10 @@ async def connect(self):
       self._queue = asyncio.Queue()
       self._listen_task = asyncio.create_task(self._listen())
       self._crypt = RiscoCrypt(self._encoding)
-      panel_id = int(await self.send_result_command('RID'))
+      try:
+        panel_id = int(await self.send_result_command('RID'))
+      except ValueError:
+        panel_id = int(await self.send_result_command('RID'), 16)
```
Comment (OnFreund) [inline pyrisco/local/risco_socket.py]:
```suggestion
      panel_id = int(await self.send_result_command('RID'), 16)
```
I think it's always hexadecimal

**Review by OnFreund — CHANGES_REQUESTED** [review]

File: pyrisco/local/panels.py (hunk 1 above)
Comment (maartenweyns) [reply pyrisco/local/panels.py]: Yes, of course. Totally missed that 😅

File: pyrisco/local/panels.py (hunk 3 above)
Comment (maartenweyns) [reply pyrisco/local/panels.py]: Yes, it looks like I was getting confused with the `rp` and `rw` naming schemes 😅

File: pyrisco/local/risco_socket.py (hunk 4 above)
Comment (maartenweyns) [reply pyrisco/local/risco_socket.py]: Okay, I will change it. I did it in a try/catch block to not break systems that work with this version

**Review by OnFreund — APPROVED** [review]
Thanks @maartenweyns!

**Pull request merged**